src/ralph/models/edx/server.py

```diff
@@ -2,12 +2,20 @@
 
 import json
 from pathlib import Path
+from typing import Literal
 
 from pydantic import root_validator, validator
 
 from ralph.models.selector import LazyModelField, selector
 
-from .base import BaseEventModel
+from .base import BaseEventModel, BaseModelWithConfig
+
+
+class ServerEventField(BaseModelWithConfig):
+    """Represents the `event` field of the ServerEventModel."""
+
+    GET: dict
+    POST: dict
```

Comment:
```suggestion
from .base import BaseEventModel
```

This part was removed in the previous commit (validate command)

Reply:
Oops I wrongly fixed the conflict... Change it now